Makefile

```diff
@@ -0,0 +1,18 @@
+container:
+	docker build . -t picl-builder:latest
+
+container_run:
+	docker run --rm -e TARGET=all -v ${PWD}:/app -v /dev:/dev --privileged picl-builder:latest
+
+container_clean:
+	docker rmi picl-builder:latest
+
+image:
+	K3OS_VERSION=v0.20.7-k3s1r0 RASPBERRY_PI_FIRMWARE=latest ./build-image.sh all
```

Comment:
Should we remove these constants here, since `build-image.sh` should work without them and use its own known-good values?

Reply:
absolutely